src/zope/applicationcontrol/tests/test_runtimeinfo.py

```diff
@@ -119,14 +142,56 @@ def test_Uptime(self):
 
         # get the uptime the current implementation calculates
         test_uptime = runtime_info.getUptime()
+        self.assertAlmostEqual(asserted_uptime, test_uptime, delta=time_tolerance)
 
-        self.assertTrue(abs(asserted_uptime - test_uptime) < time_tolerance)
+    @unittest.skipIf(not sys.path, "Need entries on path")
```

Comment:
> We're providing false assurance in many cases, since we're not enabling branch coverage tracking

True. And I've certainly been guilty of refactoring:
```python
if thing:
   return other_thing()
else:
   return default()
```
into
```python
return other_thing() if thing else default()
```
to get to that coveted 100% coverage report. (But only in cases where the non-covered branch was trivial, I promise.)

> I just wanted to understand your motivation.

An over-abundance of misplaced caution?